pkgs/development/tools/misc/kimg/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, asciidoc, imagemagick7Big, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "kimg";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "KnightOS";
+    repo = "kimg";
+    rev = version;
+    sha256 = "00gj420m0jvhgm8kkslw8r69nl7r73bxrh6gqs2mx16ymcpkanpk";
+  };
+
+  nativeBuildInputs = [ cmake asciidoc pkgconfig imagemagick7Big ];
```

Comment:
Sounds like imagemagick should go to regular buildInputs.

Reply:
It is used with `asciidoc` to generate the man pages, so it should be in `nativeBuildInputs`.

Reply:
Scratch that. I misremember, has been a week since I looked at the derivation. It does use `imagemagic` in the actual program, so this should indeed be in `buildInputs`.